package.json

```diff
@@ -52,6 +52,7 @@
     "lint:js": "eslint .",
     "start": "node server/server",
     "test": "mocha test/unit && mocha test/server",
+    "test-browser": "watchify test/frontend/frontend.bundle.js -o test/frontend/bundle.js -d",
```

Comment:
if "test/frontend/bundle.js" is compiled/browserified, we may want to add it to .gitignore and nuke the file below.